source/diagnostics.tex

```diff
@@ -2408,6 +2412,56 @@
 Equivalent to: \tcode{return os << to_string(st);}
 \end{itemdescr}
 
+\rSec3[stacktrace.format]{Formatting support}
+
+\begin{itemdecl}
+template<> struct formatter<stacktrace_entry>;
+\end{itemdecl}
+
+%FIXME: We typically use \fmtgrammarterm{} and \fmtnontermdef{} for these,
+% but they're not defined until we read utilities.tex.
```

Comment:
Then, please move those to macros.tex in this commit.
Add a comment that those two macros are used for the grammar of std::format format specifications.

Reply:
ok, done